docs/variants/msi_z690/gpu-hcl.md

```diff
@@ -0,0 +1,32 @@
+# GPU HCL
+
+## Contributing
+
+Feel free to test different graphics cards and report your results via
+[email](mailto:contact@dasharo.com) or submit a Pull Request to
+[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
+[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+
+## HCL list
+
+GPU Hardware Compatibility List presents the GPUs tested and verified
+to work with Dasharo by community. The following list does not include GPU
+which is tested and verfied in 3mdeb laboratory - this information might be
+found in [Hardware Matrix](../hardware-matrix.md) documentation.
+
+Legend:
+* GPU name - the full name of GPU including vendor and model name.
+* Memory size - total amount of GPU memory declared by vendor.
+* Memory type - GPU's type of memory.
+* Bandwith - GPU's bandwith.
+* PCI-E Architecture - declared by producer generation of PCI-E architecture.
+* Multi-Graphics Technology - information about support for Multi-Graphics
+    Technology.
+
+Information about GPU might be read from GPU package or documentation.
+
+| GPU name         | Memory size | Memory type | Bandwidth | PCI-E Architecture | Multi-Graphics Technology | Results |
+|:----------------:|:-----------:|:-----------:|:---------:|:------------------:|:-------------------------:|:-------:|
+| Nvidia GeForce GTX 1060 | 3072 MB | GDDR5 | x16       | Gen3               | 1                         | [link][1]  |
+
+[1]: https://cloud.3mdeb.com/index.php/s/JRYxxxAe7fcCczx
```

Comment:
a/a

Reply:
What solution would you suggest? Maybe creating a report on `dasharo-issues`?

Reply:
OK, improved